cura/PrinterOutput/Models/ExtruderConfigurationModel.py

```diff
@@ -4,7 +4,7 @@
 
 from PyQt5.QtCore import pyqtProperty, QObject, pyqtSignal
 
-from cura.PrinterOutput.MaterialOutputModel import MaterialOutputModel
+from cura.UI.MaterialOutputModel import MaterialOutputModel
```

Comment:
Shouldn't this also be moved back to printerOutput? It's specific to that bit of the code, right?

Reply:
Moved it back in 8780fce7a721fb0869597eabc306ff3f2bf173e1. Missed it earlier.